baseline.yaml

```diff
@@ -649,6 +649,32 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-17   
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project MUST perform a security risk assessment to 
+       understand the most likely and impactful potential security problems 
+       that could occur within the software.
+    objective: |
+       Projects need to document know threats and 
```

Comment:
```suggestion
       Projects need to document threats and 
```

Reply:
It's not clear how you document unknown threats :-). Let's omit unnecessary words.